### PR TITLE
CS-5678: create defaoult oauth client

### PR DESCRIPTION
**Pull request by ahilles107**

### CODE REVIEW

File: newscoop/install/index.php [hunk 1]
```diff
@@ -344,6 +344,7 @@
     $app['finish_service']->generateProxies();
     $app['finish_service']->installAssets();
     $app['finish_service']->saveInstanceConfig($app['session']->get('main_config'), $app['db']);
+    $app['finish_service']->createDefaultOauthCleint();
```
Comment (takeit) [inline newscoop/install/index.php]: typo: `Cleint`, should be `Client`

File: newscoop/library/Newscoop/Installer/Services/FinishService.php [hunk 2]
```diff
@@ -62,6 +62,26 @@ public function generateProxies()
     /**
      * Reload themes reditions in datbase
```
Comment (takeit) [inline newscoop/library/Newscoop/Installer/Services/FinishService.php]: wrong function's description

File: newscoop/library/Newscoop/Tools/Console/Command/CreateOAuthClientCommand.php [hunk 3]
```diff
@@ -29,7 +29,8 @@ protected function configure()
             ->addArgument('name', InputArgument::REQUIRED, 'Client name')
             ->addArgument('publication', InputArgument::REQUIRED, 'Publication alias')
             ->addArgument('redirectUris', InputArgument::REQUIRED, 'Redirect uris')
-            ->addOption('test', null, InputOption::VALUE_NONE, 'If set it will create test client with predefnied data (for automatic tests)');
+            ->addOption('test', null, InputOption::VALUE_NONE, 'If set it will create test client with predefnied data (for automatic tests)')
+            ->addOption('default', null, InputOption::VALUE_NONE, 'If set it will create default client with predefnied name');
```
Comment (takeit) [inline newscoop/library/Newscoop/Tools/Console/Command/CreateOAuthClientCommand.php]: should be `predefined`

File: newscoop/library/Newscoop/Tools/Console/Command/CreateOAuthClientCommand.php [hunk 4]
```diff
@@ -59,6 +61,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $client->setTrusted(true);
         }
 
+        if ($input->getOption('default')) {
```
Comment (takeit) [inline newscoop/library/Newscoop/Tools/Console/Command/CreateOAuthClientCommand.php]: what about check when default oauth client exist ?

Comment (ahilles107) [inline newscoop/library/Newscoop/Tools/Console/Command/CreateOAuthClientCommand.php]: i did it before creating client.

**Comment by takeit** [comment]
:+1:

**Pull request merged**